web/src/ui_init.js

```diff
@@ -692,6 +693,7 @@ export function initialize_everything() {
     settings_sections.initialize();
     settings_toggle.initialize();
     about_zulip.initialize();
+    top_left_corner.initialize();
```

Comment:
Let's put this just before `stream_list.initialize`; it commutes with basically everything here, but that's where we initialize the adjacent part of the UI.